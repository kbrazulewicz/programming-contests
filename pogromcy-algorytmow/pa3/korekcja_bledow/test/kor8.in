6
a 000111
b 000111000000
c 111111000000
d 000000111111
e 111111000000000000111111
f 000000000000111111111111
3
111111000000000000111111111111000000000000111111111111000000000000111111111111000000000000111111111111000000000000111111111111000000000000111111111111000000000000111111111111000000000000111111111111000000000000111111111111000000000000111111111111000000000000111111111111000000000000111111
111111000000000000111111111111000000000000111111111111000000000000111111111111000000000000111111111111000000000000111111111111000000000000111111111111000000000000111111111111000000000000111111111111000000000000111111111111000000000000111111111111000000000000111111000111000000111111111111000000000000111111000111000000111111111111000000000000111111000111000000111111111111000000000000111111000111000000111111111111000000000000111111000111000000111111111111000000000000111111000111000000111111111111000000000000111111000111000000111111111111000000000000111111000111000000111111111111000000000000111111000
000111000111000000000000111111111111000000000000111111111111000000000000111111111111000000000000111111111111000000000000111111111111000000000000111111111111000000000000111111111111000000000000111111111111000000000000111111111111000000000000111111111111000000000000111111111111000000000000111111111111000000000000111111111111000000000000111111111111000000000000111111111111000000000000111111111111000000000000111111111111000000000000111111111111000000000000111111111111000000000000111111111111000000000000111111111111000000000000111111111111000000000000111111111111000000000000111111111111000000111111000000111111000000111111
