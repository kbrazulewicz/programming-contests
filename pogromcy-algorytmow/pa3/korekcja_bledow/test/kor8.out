ZBYT WIELE
BLAD
OK
aaffffffffffffffffffffffffddd
